code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -158,9 +158,9 @@ GLOBAL_LIST_EMPTY(TabletMessengers) // a list of all active messengers, similar
 	QDEL_LIST(stored_files)
 	Remove_Messenger()
 
-	if(inserted_disk)
+	if(istype(inserted_disk))
```

Comment:
Why istype? When is this a non-disk non-null?

Reply:
Destroy() is being called on tablets before Initialize() is, so they don't actually have a disk/pen yet.

![image](https://user-images.githubusercontent.com/53777086/197121400-184ebaf5-04e2-4de1-9e54-8649e49dfdb7.png)